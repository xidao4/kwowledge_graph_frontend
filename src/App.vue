<template>
  <a-config-provider :locale="zh_CN">
    <div id="app">
      <transition name="fade-transform" mode="out-in">
        <router-view/>
      </transition>
    </div>
  </a-config-provider>
</template>

<script>
  import zh_CN from 'ant-design-vue/lib/locale-provider/zh_CN';

  export default {
    name: 'app',
    methods:{

    },
    data() {
      return {
        zh_CN
      };
    }
  };
</script>

<style>
body{
  height: 100%;
  overflow: hidden;
}
</style>
