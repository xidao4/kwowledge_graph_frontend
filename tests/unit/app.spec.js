import { mount } from '@vue/test-utils'
import App from '@/App';

describe('App', () => {
    let wrapper,
        vm;

    beforeEach(() => {
        wrapper = mount(App);
        vm = wrapper.vm;
    });

    // 为App的单元测试增加快照（snapshot）：
    it('包含预期html结构', () => {
        expect(vm.$el).toMatchSnapshot()
    })
});
