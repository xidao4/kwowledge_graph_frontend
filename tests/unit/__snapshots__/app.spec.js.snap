// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 包含预期html结构 1`] = `
<a-config-provider
  locale="[object Object]"
>
  <div
    id="app"
  >
    <transition-stub
      mode="out-in"
      name="fade-transform"
    >
      <router-view />
    </transition-stub>
  </div>
</a-config-provider>
`;
